PrivateHeaders/APPX/ZIP.h

```diff
@@ -325,6 +372,9 @@ namespace appx {
            << "xmlns=\"http://schemas.microsoft.com/appx/2010/blockmap\" "
            << "HashMethod=\"http://www.w3.org/2001/04/xmlenc#sha256\">";
         for (const ZIPFileEntry &entry : otherEntries) {
+            if (_IsAPPXFile(entry.fileName)) {
```

Comment:
I am not sure what you mean by that, but if we have more appx files inside of the appx they are inside of AppxBlockMap.xml so I added that as well.